ts/components/ScreenHeader.tsx

```diff
@@ -2,7 +2,12 @@ import { View } from "native-base";
 import { connectStyle } from "native-base-shoutem-theme";
 import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
 import * as React from "react";
-import { Image, ImageSourcePropType, StyleSheet } from "react-native";
+import {
```

Comment:
Please, put these changes to another story/PR

Reply:
Yes, I have already done it, the pr is https://github.com/teamdigitale/io-app/pull/1431